### ISSUE TITLE
the way of ResNet-50-Basel

### ISSUE THREAD

**Issue by JoyLuo** [layumi/2016_person_re-ID#5]
hi @layumi 
    your paper and code were read carefully. and the ResNet-50-Basel in your paper is higher then others. the question is how to train the baseline. Is the resnet50 with softmaxloss finetuned in the market1501?
    and **ours** is finetune the resnet50 with siamese loss(two losses) using the market1501?
    the last question is that the training solver between ResNet-50-Basel  and ours are same or not?
    

    thanks for your kind
    luo ze

**Comment by layumi** [comment]
I only use the simple fine-tuning method with classification loss (predict 751 classes).
There are two little tricks, which may help you:
1. Do not fix the parameters in ResNet. Market1501 or CUHK03 are different from ImageNet. 
So a little change yields a better result.

2. Add dropout layer before the final fc layer.  For example, 2048->dropout->2048->751
I found dropout rate =0.75 is great.

**Comment by JoyLuo** [comment]
@layumi 
   thanks for your advises. 
   1. the map has improved  5% by following you advises，and the **pool5** feature extracted to calculate the map. now my map is 56%。
